src/Model/Config/Communication.php

```diff
@@ -123,4 +113,49 @@ protected function getChannel(string $langAbbr): string
 
         return $channels[$langAbbr];
     }
+
+    private function encodeNgCategoryPath(array $categories, string $param): string
+    {
+        $categoryPath = implode('/', array_reverse($categories));
+        $path         = sprintf('%s:%s', $param, $categoryPath);
+
+        return sprintf('filter=%s', $this->urlPlusEncodeCategoryPath($path));
+    }
+
+    private function encodeStandardCategoryPath(array $categories, string $param): string
+    {
+        $categoriesReverse = array_reverse($categories);
+        $path              = 'ROOT';
+        $value             = ['navigation=true'];
+        foreach ($categoriesReverse as $key => $category) {
+            $filterValue = sprintf("filter{$param}%s=%s", $path, $this->urlPlusEncodeCategoryPath($category));
+            if ($key >= 1) {
+                $path .= $this->urlPlusEncodeCategoryPath(urlencode('/') . $categoriesReverse[$key - 1]);
+                $filterValue = sprintf("filter{$param}%s=%s", $path, $this->urlPlusEncodeCategoryPath($category));
```

Comment:
what is happening here is basically the same as what happens in line 133, am i correct?
So its better to not repeat that.